sycl/source/detail/platform_impl.cpp

```diff
@@ -101,13 +101,13 @@ static std::vector<DevDescT> getWhiteListDesc() {
     }
 
     if (':' != *str)
-      throw sycl::runtime_error("Malformed device white list");
+      throw sycl::runtime_error("Malformed device white list", PI_INVALID_VALUE);
```

Comment:
Isn't it perfect candidate for CL_OUT_OF_RESOURCES or CL_OUT_OF_HOST_MEMORY :) ?

Reply:
I'm really happy that it is not...